lib/presentation/widget/myspace/my_space_viewmodel.dart

```diff
@@ -81,7 +84,17 @@ class MySpaceViewModel extends BaseViewModel {
       this._copyMultipleFilesToSharedSpaceInteractor,
       this._removeMultipleDocumentsInteractor,
       this._downloadMultipleFileIOSInteractor
-  ) : super(store);
+  ) : super(store) {
+    _storeStreamSubscription = store.onChange.listen((event) {
+      event.mySpaceState.viewState.fold(
+        (failure) => null,
+        (success) {
+          if (success is RemoveDocumentViewState) {
+            getAllDocument();
+          }
+      });
+    });
+  }
```

Comment:
Are you sure it's the only way to fix it ? Why not fetching documents in remove thunk action directly ?

Reply:
he tried some ways, but it is the best right now. When he did like you said, 2 actions are dispatch to store nearly, so Store connector can not react exactly. `RemoveSuccess` and `Loading` action

Reply:
Ok I see

Reply:
This works for me, am I missing smth ?
```
  ThunkAction<AppState> _removeDocumentAction(List<DocumentId> documentIds) {
    return (Store<AppState> store) async {
      await _removeMultipleDocumentsInteractor.execute(documentIds: documentIds)
          .then((result) => result.fold(
              (failure) => store.dispatch(MySpaceAction(Left(failure))),
              (success) => store.dispatch(_removeDocumentSuccessAction(success))));
    };
  }

  ThunkAction<AppState> _removeDocumentSuccessAction(Success success) {
    return (Store<AppState> store) async {
      store.dispatch(MySpaceAction(Right(success)));
      getAllDocument();
    };
  }

  void getAllDocument() {
    store.dispatch(_getAllDocumentAction());
  }
```
![ducdelete](https://user-images.githubusercontent.com/48354990/106772670-8b8ac680-6640-11eb-8a33-9293edbb461b.gif)

Reply:
@imGok you are missing the toast message when remove a file successfully